src/shared/watchdog.c

```diff
@@ -47,8 +47,21 @@ static int disable_watchdog(void) {
         return 0;
 }
 
+static int gettimeout_watchdog(usec_t *usec) {
```

Comment:
as per our usual coding style, please name return values either `ret` or `ret_xyz`, i.e. in this case, maybe `ret` or `ret_usec`

Reply:
Ok.